README.Mac.md

```diff
@@ -1,294 +1,281 @@
 ```
-mysql -u root
+brew update
+brew upgrade
+brew install httpd
+brew link httpd
+sudo brew services start httpd
 ```
 
-If the command is not found use the following 
-
+#### (useful) terminal commands for Apache
+How to start, stop, and restart apache.
 ```
-sudo sh -c 'echo /usr/local/mysql/bin > /etc/paths.d/mysql'
+sudo apachectl start
+sudo apachectl stop
+sudo apachectl -k restart
 ```
-Then restart terminal or open a new tab or window and start this step over.
 
-Once in the mysql command prompt, use the following command to reset the password
+### 1.5.1 Modify Apache Configuration
 
-``` 
-SET PASSWORD FOR 'root'@'localhost' = PASSWORD('$newpwd');
+Modify the apache configuration file for our development environment.
+```
+sudo open -e /usr/local/etc/httpd/httpd.conf
 ```
-<i>$newpwd ⇾ you new desired password for the root user</i>
-
-## 1.5 Smarty 3
-
-### 1.5.1 Obtaining Smarty 3
- 
-Go to [http://www.smarty.net/download](http://www.smarty.net/download) and download the latest stable release of Smarty 3.x.x
-
-### 1.5.2 Setting up Smarty 3
-
-Create the following directories if they do not already exist:
 
+**a.** Find & modify the ServerName to:
 ```
-sudo mkdir /usr/local/lib/php
-sudo mkdir /usr/local/lib/php/Smarty
+ServerName localhost
 ```
 
-Next, unzip the downloaded zip file and locate the libs directory.  Move it into the Smarty directory (created above), and rename it to smarty3 in the process. Use the following: 
-
+**b.** Find the lines for `DocumentRoot` and Directory in apache and change them to:
 ```
-sudo mv /path/to/Smarty-3.x.x/libs/ /usr/local/lib/php/Smarty/smarty3
+DocumentRoot "/Users/<your_user>/Development/GitHub/Loris/htdocs/"
+<Directory "/Users/<your_user>/Development/GitHub/Loris/htdocs/">
 ```
 
-After you have moved the directory ensure to add the path ```/usr/local/lib/php/Smarty``` to the ```include_path``` in the php.ini file (see [section 1.3.4](#include_path) part 3).
-
-# 2. Installing LORIS Code Base
-
-This section goes over how to obtain the LORIS code base and setup the database on your local machine.
-
-## 2.1 Obtaining the LORIS code
+**c.** In the same <Directory> block, modify AllowOverride to allow all:
+```
+# AllowOverride controls what directives may be placed in .htaccess files.
+# It can be "All", "None", or any combination of the keywords:
+#   AllowOverride FileInfo AuthConfig Limit
+#
+AllowOverride All
+```
 
-Download the latest release from the [releases page](https://github.com/aces/Loris/releases) and
-extract it to `~/Sites/$projectname`
+**d.** Enable mod_rewrite which is commented out by default:
+```
+LoadModule rewrite_module lib/httpd/modules/mod_rewrite.so
+```
 
-<i>$projectname ⇾ “loris” or one-word project name</i>
+**e.** Modify the User & Group (replace your_user again with your macOS user) to:
+```
+User your_user
+Group staff
+```
 
-## 2.2 Installing LORIS
+**f.** Modify the ServerName to:
```

Comment:
back tick around `ServerName`

Reply:
Adjusting the server name as been done on line 95+. Should keep only one of the 2.